server-side/fill-in-the-blank/scatter-plot-colab/README.md

```diff
@@ -0,0 +1,12 @@
+[BERT Difference Plots Colab](https://colab.research.google.com/drive/1xfPGKqjdE635cVSi-Ggt-cRBU5pyJNWP?usp=sharing)
+
+### Dev
+
+Hot reload JS
+
+````
+find . | entr rsync -a --omit-dir-times --no-perms --exclude node_modules --exclude .git "$PWD" demo@roadtolarissa.com:../../usr/share/nginx/html/colab/
+````
+
+- https://colab.research.google.com/notebooks/snippets/advanced_outputs.ipynb
```

Comment:
will this path be correct when submitted? should it be the github location?

Reply:
Checked in full notebook to LFS and added Dev instructions